### PR TITLE
ci fix

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
           fingerprints: "7b:8f:c8:1c:13:a3:b4:46:ec:9a:a5:0d:3f:62:69:78"
       - run: ssh-keyscan github.com >> ~/.ssh/known_hosts
       - run: git config --global user.email "sunpy@circleci" && git config --global user.name "SunPy Circle CI"
-      - run: git clone git@github.com:sunpy/sunpy-figure-tests.git --depth 1 -b sunpy-${CIRCLE_BRANCH} ~/sunpy-figure-tests/
+      - run: git clone git@github.com:sunpy/sunpy-figure-tests.git --depth 1 -b sunpy-master ~/sunpy-figure-tests/
```
Comment (nabobalis) [inline .circleci/config.yml:74]: Until the repo has been renamed at least.

**Comment by dstansby** [comment]
Can you give some context as to what this is fixing?

**Comment by nabobalis** [comment]
The failing CI builds on the cronjob.

File: .circleci/config.yml (hunk 1 above)
Comment (nabobalis) [inline .circleci/config.yml:74]:
```suggestion
      - run: git clone git@github.com:sunpy/sunpy-figure-tests.git --depth 1 -b sunpy-${CIRCLE_BRANCH}~/sunpy-figure-tests/
```

**Pull request merged**